caffe2/proto/caffe2.proto

```diff
@@ -186,6 +253,24 @@ message OperatorDef {
   // This is an optional string with no assumed characteristics as
   // operators can be constructed in any language.
   optional string debug_info = 10;
+
+  // place for additional information
+  // for example, for some training platform, if they want to tag the
+  // operator for placing strategy, component info and parallel strategy,
+  // such information is not about the model/operator itself, but about
+  // the runtime/platform.
+  // existing solution is saving extra info in device option,
+  // however, it's more natural to keep the information here
+  // existing usage: https://fb.quip.com/yAiuAXkRXZGx
```

Comment:
I think this document is not available for OSS community, which makes it a bad comment, so have some inline examples will be good.